src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -22,13 +22,15 @@ def initialize(disks, pager)
 
       # table items. See CWM::Table#items
       def items
+        probed_graph = Y2Storage::StorageManager.instance.y2storage_probed
```

Comment:
well, to have something workable to end of sprint. Is better solution to have partitioner singleton which have

1. old device graph
2. current graph
3. working copy graph

?